app/templates/components/forms/login-form.hbs

```diff
@@ -35,9 +35,13 @@
             <a href="{{href-to 'forgot-password'}}" class="text muted"> {{t 'Forgot your password ?'}}</a>
           </div>
         </div>
-        <div class="ui secondary segment center aligned">
-          <a href="{{href-to 'register'}}" class="text muted weight-800">{{t 'New to site? Create Account'}}</a>
-        </div>
+        {{#link-to 'register'}}
```

Comment:
Like I said before, a `segment` has to be the direct child of `segments`. Using link-to doesn't change that.